core/types/outputpayload/transfercrosschainfee.go

```diff
@@ -0,0 +1,61 @@
+package outputpayload
+
+import (
+	"errors"
+	"io"
+
+	"github.com/elastos/Elastos.ELA/common"
+	"github.com/elastos/Elastos.ELA/crypto"
+)
+
+type CrossChainFee struct {
+	Version byte
+	TxHash  []byte
```

Comment:
In out project, a hash is always a Uint256 type, the TxHash should be a common.Uint256 not a []byte.

Reply:
Already changed as required